_docs/release-notes.md

```diff
@@ -24,6 +24,21 @@ Visit [https://app.tidepool.org](https://app.tidepol.org) to see the results of
 
```

Comment:
I just noticed there's a type in a URL in this doc (not part of the changes): app.tidepol.org

Reply:
Good find.